File: TinyScript/js/src/main/js/parser/FunctionDefineStmt.ts
import Stmt from "./Stmt";
import ASTNode from "./ASTNode";
import ASTNodeTypes from "./ASTNodeTypes";

export default class FunctionDefineStmt extends Stmt {

    constructor(parent: ASTNode, types: ASTNodeTypes | null, label: string | null) {
        super(parent, ASTNodeTypes.FUNCTION_DECLARE_STMT, "function");
    }
}

File: TinyScript/java/src/main/java/paser/FunctionDefineStmt.java
package paser;

/**
 * @author MFine
 * @version 1.0
 * @date 2021/8/23 0:22
 **/
public class FunctionDefineStmt extends Stmt {
    protected FunctionDefineStmt(ASTNode parent) {
        super(parent, ASTNodeTypes.FUNCTION_DECLARE_STMT, "function");
    }
}
